felles/src/main/kotlin/no/nav/familie/kontrakter/felles/tilbakekreving/OpprettTilbakekrevingRequest.kt

```diff
@@ -0,0 +1,14 @@
+package no.nav.familie.kontrakter.felles.tilbakekreving
+
+import no.nav.familie.kontrakter.felles.PersonIdent
+
+data class OpprettTilbakekrevingRequest(val fagsystem: String,
+                                        val eksternFagsakId: String,
+                                        val personIdent: PersonIdent,
```

Comment:
personIdent er vel ganske innarbeidet begrep både for EF og Baks, så foreslår at vi lar den være som den er. Men kan si meg enig i at objekter som kun inneholder 1 felt ikke gir noe merverdi utenom kanskje å sikre samme begrepsbruk på tvers.

Reply:
Ellers i kontrakter er `ident` og `fnr` mest brukt, kun noen få tilfeller med `personIdent`.
Ser jeg selv ikke har vært helt konsekvent med ident/fnr i noen DTOer der

Reply:
Enig i at `personIdent` favner bredere/bedre (fnr, dnr + ny dollyvariant for preprod), men denne kan være en `String` for å unngå unødvendig wrapping.

Reply:
(4 av 4 i en spørreundersøkelse på EF stemte for `personIdent`) 😂

Reply:
Jeg kommer endre om noen objekter som ikke helt brukes ennå til å bruke personIdent 👍